packages/sdk/src/dataProtector/sharing/subgraph/getCollectionById.ts

```diff
@@ -0,0 +1,38 @@
+import { gql, type GraphQLClient } from 'graphql-request';
+import type { Address } from '../../types/index.js';
+
+export async function getCollectionById({
+  graphQLClient,
+  collectionTokenId,
+}: {
+  graphQLClient: GraphQLClient;
+  collectionTokenId: Address;
+}) {
+  const getProtectedDataQuery = gql`
+    query {
+      collection(id: "${collectionTokenId}") {
+        id
+        owner {
+          id
+        }
+        protectedDatas {
+          id
+        }
+        subscriptionParams {
+          duration
+          price
+        }
+      }
+    }
+  `;
+  const { collection } = await graphQLClient.request<{
+    collection: {
+      id: Address;
```

Comment:
```suggestion
      id: bigint;
```

Reply:
no it's a hexadecimal. It's not a bigInt for me

Reply:
Oh yes, string then, all good